ops/op2/mod.rs

```diff
@@ -185,6 +187,15 @@ fn generate_op2(
   };
   let is_async = signature.ret_val.is_async();
   let is_reentrant = config.reentrant;
+  let meta_token_stream: TokenStream = config
+    .meta
+    .iter()
+    .map(|(key, value)| {
+      quote! {
+          #key: #value,
```

Comment:
We actually want the output to look something like this (a `const` `HashMap`). You should probably create something that looks like this (the `HashMap` bits will probably need to be in the `DECL` below.

```
::std::collections::HashMap<&str, &str> = [

#( ( #key, #value ), )*

].iter().cloned().collect();
```